packages/devtools/pubspec.yaml

```diff
@@ -44,7 +44,8 @@ dependencies:
 
 dev_dependencies:
   build_runner: ^1.3.0
-  build_web_compilers: ^1.2.0
+  build_test: ^0.10.0
+  build_web_compilers: '>=1.2.0 <3.0.0'
```

Comment:
Fwiw, this I believe was the cause of the failure (2.x uses kernel based ddc instead of analyzer based ddc)

Reply:
Thanks! I presume all the other changes should work without this, so I can probably split that bit out but land the rest if it's causing issues?

Reply:
Thanks, I backed that out since  it still failed. If this goes green we can land the rest.

I'll also do some debugging with the new version (probably via Travis if I can't get it to repro locally).